Update MySQL docs to reflect timeout variable changes in version 8.0.26+

vai-airbyte opened the pull request:
- Added version-specific guidance for MySQL timeout variables:
  - `replica_net_timeout` for MySQL 8.0.26 and later
  - `slave_net_timeout` for earlier versions
- Highlighted the deprecation of `slave_net_timeout` and provided clear, version-aware instructions.

Reference: https://dev.mysql.com/doc/mysql-replication-excerpt/8.0/en/replication-options-replica.html#sysvar_replica_net_timeout

vai-airbyte committed:
Update MySQL docs to reflect timeout variable changes in verison 8.0.26+

- Added version-specific guidance for MySQL timeout variables:
  - `replica_net_timeout` for MySQL 8.0.26 and later
  - `slave_net_timeout` for earlier versions
- Highlighted the deprecation of `slave_net_timeout` and provided clear, version-aware instructions.

Reference: https://dev.mysql.com/doc/mysql-replication-excerpt/8.0/en/replication-options-replica.html#sysvar_replica_net_timeout

vai-airbyte committed:
Apply suggestions from code review

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

vai-airbyte committed:
Fix formatting for note on MySQL timeout variable

vai-airbyte committed:
Apply suggestions from code review

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

ian-at-airbyte reviewed (approved):
Looks good. Left two very minor comments for consideration.

vai-airbyte committed:
Clarify MySQL timeout settings in troubleshooting guide

Updated MySQL troubleshooting instructions for timeout settings, clarifying the use of 'replica_net_timeout' and 'slave_net_timeout' based on MySQL version.

vai-airbyte commented on docs/integrations/sources/mysql/mysql-troubleshooting.md:
````diff
@@ -81,11 +81,23 @@ When a sync runs for the first time using CDC, Airbyte performs an initial consi
 
+**For MySQL 8.0.26 and later:**
+
+```
````
Thanks Ian, actually they're SQL settings so I added ```sql.

vai-airbyte committed:
Apply suggestion from @github-actions[bot]

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

vai-airbyte committed:
Apply suggestion from @github-actions[bot]

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

vai-airbyte reviewed (commented):
Implemented suggestions.
